client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementPublicationAndVoting.vue

```diff
@@ -43,7 +43,7 @@ All rights reserved
       :has-permission-to-edit="editable && statement.attributes.isManual"
       :translation-keys="translationKeys"
       ref="listComponent"
-      @reset="resetForm()"
```

Comment:
We still need this for resetting the form

Reply:
Done in [2b73d01](https://github.com/demos-europe/demosplan-core/pull/3996/commits/2b73d01f05031a3bec8aa63796a94e41eddae79d)